src/io/flutter/samples/FlutterSampleManager.java

```diff
@@ -38,7 +38,7 @@
 
 public class FlutterSampleManager {
 
-  private static final String SNIPPETS_REMOTE_INDEX_URL = "https://docs.flutter.io/snippets/index.json";
+  private static final String SNIPPETS_REMOTE_INDEX_URL = "BROKEN_https://docs.flutter.io/snippets/index.json";
```

Comment:
Can you clean this up? I think we may display this URL in the UI?
Also, does this patch work for Stable as well as master?

Reply:
I'm pretty sure the URL isn't displayed.

I don't know if this works for stable.

After reading create.dart (flotter_tools) I'm thinking we might have to live with this problem for this release. The right way to fix it is to invoke `flutter create --list-samples`.

No one has hit this problem even though it has existed for weeks. I'm no longer convinced fixing it is important.